boa_engine/src/context/mod.rs

```diff
@@ -550,6 +552,21 @@ impl<'host> Context<'host> {
     pub fn job_queue(&self) -> MaybeShared<'host, dyn JobQueue> {
         self.job_queue.clone()
     }
+
+    /// Get the [`RuntimeLimits`].
+    pub fn runtime_limits(&mut self) -> RuntimeLimits {
+        self.vm.runtime_limits
+    }
```

Comment:
Do we need a mutable reference to `self` here? Wouldn't it be enough with a non-mutable reference?